### ISSUE TITLE
A11y: Sliders - Make setting "ariaLive" controllable in the backend

### ISSUE THREAD

**Issue by tieupmedia** [shopware/platform#5227]
For product / image slider cms elements (in the shopping experiences) a config option (checkbox) should be added, to enable/disable the `ariaLive` setting form within the backend. 

Relates to https://github.com/shopware/shopware/blob/trunk/changelog/release-6-6-4-0/2024-05-31-remove-unwanted-aria-live-attributes-from-sliders.md

**Comment by lasomethingsomething** [comment]
Resolved in 6.6.10.0, so closing. Thanks for filing this!
